packages/xo-web/src/common/intl/messages.js

```diff
@@ -1014,6 +1014,7 @@ const messages = {
   poolNeedsDefaultSr: 'The pool needs a default SR to install the patches.',
   vmsHaveCds: '{nVms, number} VM{nVms, plural, one {} other {s}} {nVms, plural, one {has} other {have}} CDs',
   ejectCds: 'Eject CDs',
+  disabledInstallPatchesTooltip: 'High Availability should be disabled',
```

Comment:
IMO, the ID of the message should represent the meaning of the message, not where it is in XO.